webapi/src/main/scala/org/knora/webapi/messages/v2/responder/valuemessages/ValueMessagesV2.scala

```diff
@@ -2482,13 +2483,13 @@ object IntervalValueContentV2 extends ValueContentReaderV2[IntervalValueContentV
     val intervalValueHasStart: BigDecimal = jsonLDObject.requireDatatypeValueInObject(
       key = OntologyConstants.KnoraApiV2Complex.IntervalValueHasStart,
       expectedDatatype = OntologyConstants.Xsd.Decimal.toSmartIri,
-      validationFun = stringFormatter.validateBigDecimal
+      validationFun = (s, errerFun) => ValuesValidator.validateBigDecimal(s).getOrElse(errerFun)
```

Comment:
```suggestion
      validationFun = (s, errorFun) => ValuesValidator.validateBigDecimal(s).getOrElse(errorFun)
```

Please also check the other occurences.